ios/packages/test-utils/Sources/ui-test/AssetUITestCase.swift

```diff
@@ -51,7 +51,16 @@ open class AssetUITestCase: XCTestCase {
      */
     open func openFlow(_ mockName: String) {
         navigateToAssetCollection()
-        app.otherElements.buttons[mockName].firstMatch.tap()
+
+        let button = app.otherElements.buttons[mockName].firstMatch
+        app.scrollToElement(element: button)
+
+        waitFor(button)
+        button.tap()
```

Comment:
this should be moved into our actual UItests by overriding `navigateToAssetCollection`, since the `SegmentControlView` should be in the demo app